tests/unit/data/endpoints/valid-rules/array-index.json

```diff
@@ -0,0 +1,45 @@
+{
+  "version": "1.3",
+  "parameters": {
+    "ResourceList": {
+      "type": "stringArray"
+    }
```

Comment:
Related to my pervious comment: We should included tests for string array parameters that define default values.